addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -113,37 +113,10 @@ var SnippetEditor = Widget.extend({
         // Initialize move/clone/remove buttons
         if (this.isTargetMovable) {
             this.dropped = false;
-            const smoothScrollOptions = this.options.getScrollOptions({
-                jQueryDraggableOptions: {
-                    cursorAt: {
-                        left: 10,
-                        top: 10
-                    },
-                    handle: '.o_move_handle',
-                    helper: () => {
-                        var $clone = this.$el.clone().css({width: '24px', height: '24px', border: 0});
-                        $clone.appendTo(this.$el[0].ownerDocument.body).removeClass('d-none');
-                        return $clone;
-                    },
-                    start: this._onDragAndDropStart.bind(this),
-                    stop: (...args) => {
-                        // Delay our stop handler so that some wysiwyg handlers
-                        // which occur on mouseup (and are themself delayed) are
-                        // executed first (this prevents the library to crash
-                        // because our stop handler may change the DOM).
-                        setTimeout(() => {
-                            this._onDragAndDropStop(...args);
-                        }, 0);
-                    },
-                    refreshPositions: true, // So the dropzone expands when its size increases.
-                },
-            });
-            const modalAncestorEl = this.$target[0].closest('.modal');
-            const $scrollable = modalAncestorEl && $(modalAncestorEl)
-                || (this.options.wysiwyg.snippetsMenu && this.options.wysiwyg.snippetsMenu.$scrollable)
-                || (this.$scrollingElement.length && this.$scrollingElement)
-                || $().getScrollingElement(this.ownerDocument);
-            this.draggableComponent = new SmoothScrollOnDrag(this.$el, $scrollable, smoothScrollOptions);
+            this.draggableComponent = this._initSmoothScrollOnDrag(".o_move_handle", this.$el);
+            if (!this.$target[0].matches("section")) {
```

Comment:
It will be nice if you add a comment here to explain a little bit.
for instance: "If it is not a `<section>` (so an inner element), we allow the user to move it by grabbing the image inside" or something like that ;)